test/runtime/lb.go

```diff
@@ -260,13 +275,25 @@ tc qdisc del dev lbtest2 clsact 2> /dev/null || true
 tc qdisc add dev lbtest2 clsact
 tc filter add dev lbtest2 ingress bpf da obj tmp_lb.o sec from-netdev
 `
-	err := helpers.RenderTemplateToFile("create_veth_interface", script, 0777)
+	scriptName := "create_veth_interface"
+	log.Infof("generating veth script: %s", scriptName)
+	err := helpers.RenderTemplateToFile(scriptName, script, os.ModePerm)
 	if err != nil {
 		return err
 	}
-	path := "/vagrant/create_veth_interface"
-	res := node.Exec("sudo ip addr add fd02:1:1:1:1:1:1:1 dev cilium_host")
-	res.ExpectSuccess()
-	node.Exec(fmt.Sprintf("sudo %s", path))
-	return os.Remove(path)
+
+	// filesystem is mounted at path /vagrant on VM
+	scriptPath := fmt.Sprintf("%s%s", helpers.BasePath, scriptName)
+
+	ipAddrCmd := "sudo ip addr add fd02:1:1:1:1:1:1:1 dev cilium_host"
+	res := node.Exec(ipAddrCmd)
```

Comment:
Does this also check the output? At a glance it looks like the `ExpectSuccess()` was dropped, which may allow the execution to continue despite a failure.

Reply:
If you run the test multiple times locally, the command will fail. I can't see why it would ever not fail in the context of the tests, so I decided to remove it.